test/Conversion/CalyxToFSM/lower-seq.mlir

```diff
@@ -31,7 +31,7 @@
 // CHECK-NEXT: }
       
 
-calyx.program "main" {
+module attributes {calyx.entrypoint = "main"} {
```

Comment:
These `calyx.entrypoint` attributes shouldn't be necessary for the `CalyxToFSM` passes to succeed.

Reply:
Yeah it's not required as far as I know. Should I leave them out of the test cases?

Reply:
I'd do that - tests should be as stripped as possible.